### PR TITLE
Sean Hughes - completed javascript-loops

### PR DESCRIPTION
**Pull request by Sean-Hughes-1**
image

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
```
Comment (JenBowman1) [inline javascript-loops/main.js:11]: Hey Sean, you are creating 2 counter variables here : 'i' and 'currentNumber'. Please only use currentNumber to iterate through the loop.

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    numbers.push(currentNumber);
+    currentNumber++;
+    i++;
+  }
+  return numbers;
+}
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  var i = 0;
```
Comment (JenBowman1) [inline javascript-loops/main.js:23]: Please only use currentNumber to iterate through the loop, an additional 'i' variable shouldn't be needed here.

**Review by JenBowman1 — CHANGES_REQUESTED** [review]
Hey Sean, looks great! Just a few small changes.

**Comment by Sean-Hughes-1** [comment]
changes made

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**